### PR TITLE
FIX: Increase FSL serialization precision

### PR DESCRIPTION
**Pull request by effigies**
Was reviewing when you merged. :-)

### CODE REVIEW

File: nitransforms/io/fsl.py [hunk 1]
```diff
@@ -46,14 +46,14 @@ def from_ras(cls, ras, moving=None, reference=None):
 
         # Adjust for reference image offset and orientation
         refswp, refspc = _fsl_aff_adapt(reference)
-        pre = reference.affine.dot(inv(refspc).dot(inv(refswp)))
+        pre = reference.affine @ inv(refswp @ refspc)
```
Comment (effigies) [inline nitransforms/io/fsl.py:49]: I think it makes sense to think of the matrices as `aff` and `swp @ spc`. So you are always either `aff @ inv(swp @ spc)` or `swp @ spc @ inv(aff)`.

File: nitransforms/io/fsl.py [hunk 2]
```diff
@@ -46,14 +46,14 @@ def from_ras(cls, ras, moving=None, reference=None):
...
 
         # Adjust for moving image offset and orientation
         movswp, movspc = _fsl_aff_adapt(moving)
-        post = inv(movswp).dot(movspc.dot(inv(moving.affine)))
+        post = movswp @ movspc @ inv(moving.affine)
```
Comment (effigies) [inline nitransforms/io/fsl.py:53]: That catches that here you had `movswp` inverted and `movspc` not.

**Review by oesteban — APPROVED** [review]
Thanks for this. I've updated the test files so that the new precision is honored.

File: nitransforms/tests/utils.py [hunk 3]
```diff
@@ -24,4 +24,4 @@ def assert_affines_by_filename(affine1, affine2):
     else:
         xfm1 = np.loadtxt(str(affine1))
         xfm2 = np.loadtxt(str(affine2))
-        np.testing.assert_almost_equal(xfm1, xfm2)
+        np.allclose(xfm1, xfm2, atol=1e-04)
```
Comment (effigies) [inline nitransforms/tests/utils.py:27]: We just lost an assertion.

**Pull request merged**